unittests/test-contracts/payloadless/payloadless.cpp

```diff
@@ -5,3 +5,46 @@ using namespace eosio;
 void payloadless::doit() {
    print("Im a payloadless action");
 }
+
+constexpr size_t cpu_prime_max = 15375u;
+
+bool is_prime(int p) {
+   if (p == 2) {
+      return true;
+   } else if (p <= 1 || p % 2 == 0) {
+      return false;
+   }
+
+   bool prime = true;
+   const int to = sqrt(p);
+   int i;
+   for (i = 3; i <= to; i += 2) {
+      if (!((prime = bool(p)) % i)) break;
```

Comment:
How does it work?  I'm thinking the prime calc somewhere in here should be `p % i`... setting `prime = bool(p)` makes prime always true as long as `p > 0` but I don't see how you get the needed` p % i` out of this

Reply:
I believe it is supposed to be `prime = bool(p % i)`?